nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -952,7 +952,27 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 			// The object has no text, but we do want to render its children.
 			auto [varChildren, accChildRes] = getAccessibleChildren(pacc, 0, childCount);
 			if (S_OK != accChildRes || varChildren.size() == 0) {
-				LOG_ERROR(L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes);
+				std::wstringstream msg;
+				msg << L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes;
+				switch (accChildRes) {
+				case E_NOINTERFACE:
+					msg << L" (E_NOINTERFACE, No such interface supported)";
+					// This is certainly an error, and indicates a bug in the IA2 provider.
+					LOG_ERROR(msg.str());
+					break;
+				case CO_E_OBJNOTCONNECTED:

```

Comment:
Ah. The code COM is returning in Firefox is actually RPC_E_DISCONNECTED, not CO_E_OBJNOTCONNECTED. You should handle both the same way, though; i.e. log as warnings.

Reply:
Thanks, I've updated the description with this now.